### PR TITLE
Seed file, tota11y fixes and small changes

### PR DESCRIPTION
**Pull request by tejasraibagi26**

### CODE REVIEW

File: public/js/login.js [hunk 1]
```diff
@@ -53,9 +53,14 @@
     loading.show();
     button.hide();
 
-    $.post("/user/login", data, function (data) {
+    //console.log(window.location.href.split("3000")[1]);
+    let route = window.location.href.split("3000")[1];
+    $.post(route, data, function (data) {
+      console.log(data);
```
Comment (troytomasch) [inline public/js/login.js:59]: Remove console.log

File: routes/feed.js [hunk 2]
```diff
@@ -35,24 +34,36 @@ router.get("/", async (req, res) => {
...
+    } catch (error) {
+      res.json(error);
+    }
     venues.push(venueData);
   }
   for (let i = 0; i < activities.length; i++) {
     let bookingData = await booking.getBookingById(activities[i].bookingId);
     bookings.push(bookingData);
   }
+  console.log(activities);
```
Comment (troytomasch) [inline routes/feed.js:49]: Remove console.log

File: tasks/seed.js [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+      console.log(`Created Report for ${type} with id ${addReport.id}`);
+    }
+  }
+
+  console.log("Closing DB Connection");
+  await db.serverConfig.close();
+  console.log("Done");
+};
+
+main().catch((err) => console.log(err));
```
Comment (troytomasch) [inline tasks/seed.js:572]: Is it possible to return or something so that the file finishes running and doesn't hang at the end?

**Review by troytomasch — CHANGES_REQUESTED** [review]

File: tasks/seed.js (hunk 3 above)
Comment (troytomasch) [reply tasks/seed.js:572]: Also, resolve conflicts

**Review by troytomasch — APPROVED** [review]

File: tasks/seed.js (hunk 3 above)
Comment (tejasraibagi26) [reply tasks/seed.js:572]: The db wont close for some reason. I even downgraded to 3.6.9

**Pull request merged**